modules/zivid/experimental/projection.py

```diff
@@ -0,0 +1,145 @@
+
+    The image projection will stop either when the instance is destroyed, when the stop() method is called,
+    or when exiting the "with" block if used as a context manager.
+    """
+
+    def __init__(self, impl):
+        """Initialize ProjectedImage wrapper.
+
+        This constructor is only used internally, and should not be called by the end-user.
+
+        Args:
+            impl: Reference to internal/back-end instance.
+
+        Raises:
+            TypeError: If argument does not match the expected internal class.
+        """
+        if isinstance(impl, _zivid.ProjectedImage):
+            self.__impl = impl
+        else:
+            raise TypeError(
+                "Unsupported type for argument impl. Got {}, expected {}.".format(
+                    type(impl), type(_zivid.ProjectedImage)
+                )
+            )
+
+    def __str__(self):
+        return str(self.__impl)
+
+    def capture(self, settings2d):
+        """Capture a single 2D frame without stopping the ongoing image projection.
+
+        This function can only be used with a zero-brightness 2D capture, otherwise it
+        will interfere with the projected image. An exception will be thrown if settings
+        contains brightness > 0 or if brightness is not set at all.
+
+        Args:
+            settings2d: A Settings2D instance to be used for 2D capture.
+
+        Returns:
+            A Frame2D containing a 2D image plus metadata.
+
+        Raises:
+            TypeError: If argument is not a Settings2D.
+        """
+
+        if isinstance(settings2d, Settings2D):
+            return Frame2D(self.__impl.capture(_to_internal_settings2d(settings2d)))
+        raise TypeError("Unsupported settings type: {}".format(type(settings2d)))
+
+    def stop(self):
+        """Stop the ongoing image projection."""
+        self.__impl.stop()
+
+    def active(self):
+        """Check if a handle is associated with an ongoing image projection.
+
+        Returns:
+            A boolean indicating projection state.
+        """
+        return self.__impl.active()
+
+    def release(self):
+        """Release the underlying resources and stop projection."""
+        try:
+            impl = self.__impl
+        except AttributeError:
+            pass
+        else:
+            impl.release()
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exception_type, exception_value, traceback):
+        self.release()
+
+    def __del__(self):
+        self.release()
+
+
+def projector_resolution(camera):
+    """Get the resolution of the internal projector in the Zivid camera.
+
+    Args:
+        camera: The Camera instance to get the projector resolution of.
+
+    Returns:
+        The resolution as a tuple (height,width).
+    """
+    return _zivid.projection.projector_resolution(
+        camera._Camera__impl  # pylint: disable=protected-access
+    )
+
+
+def show_image(camera, image_bgra):
```

Comment:
In the future in the C++ API we may add overloads for rgba, rgb etc. Then we will get a bit hurt here, since there is no type information passed along with image_bgra. Anyway that is a problem for later.

Reply:
I could rename it to `show_image_bgra`. What do you think?

Reply:
Or leave it like this since it's Experimental.

Reply:
I tried to find some other places in python sdk to follow as convention. We don't have anything exactly like this in the SDK (a variadic type as an input).

For point_cloud.copy_data it looks like ` point_cloud.copy_data(data_format="xyzrgba")`. Frame2D has two getters `frame_2d.image_rgba()` and `frame_2d.image_bgra()`. 

The most direct mapping to C++ SDK would be to have a this accept a  Image wrapper type. I do see that we have Image class, so the most C++ comparable fix would be to add a way to make an zivid-python Image, similar to what we added in C++ in 2.10. https://github.com/zivid/zivid-python/blob/36fd451396e3c92935bc8c9a84eb7be9a610c004/modules/zivid/image.py But we would get some complexity since as of now the C++ API will only accept a `Image<ColorBGRA>`. 

I think `show_image_bgra` and having it like you have it now is okey, then we can improve on this when we at some point add more Image types for showImage in the C++ API, without breaking too much existing zivid-python code.

Reply:
Renamed here:
https://github.com/zivid/zivid-python/pull/234/commits/72854f750e707060925abc4af4819a5ca0995a17